buildnotifylib/server_configuration_dialog.py

```diff
@@ -16,18 +16,25 @@ def __init__(self, url, conf, parent=None):
         QDialog.__init__(self, parent)
         self.ui = Ui_serverConfigurationDialog()
         self.ui.setupUi(self)
-        self.ui.addServerUrl.setText(url)
+
+        if url is not None:
+            self.ui.addServerUrl.setText(url)
+
         self.conf = conf
         self.parent = QtGui.QStandardItem("All")
         all_timezones = [Config.NONE_TIMEZONE]
         all_timezones.extend(pytz.all_timezones)
         self.ui.timezoneList.addItems(all_timezones)
-        self.server = conf.get_server_config(url)
-        self.ui.timezoneList.setCurrentIndex(all_timezones.index(self.server.timezone))
-        self.ui.displayPrefix.setText(self.server.prefix)
+
+        if url is not None:
+            self.server = conf.get_server_config(url)
```

Comment:
Can we move the `self.ui.addServerUrl.setText(url)` from line 21 to this conditional ?